src/interfaces/express.ts

```diff
@@ -65,15 +62,13 @@ export function makeExpressHandler(
 
     populateErrorMessage(err, em);
 
-    if (isObject(client) && isFunction(client.sendError)) {
-      client.sendError(em);
-    }
+    if (is.object(client) && is.function(client.sendError))
+      { client.sendError(em); }
```

Comment:
Syntax here looks a bit messed up

Reply:
The syntax is fine, it's `clang-format` doing *weird* stuff. It still runs fine.